ui/pages/asset/components/asset-options.js

```diff
@@ -85,14 +85,39 @@ const AssetOptions = ({
   );
 };
 
+const isntFunc = (p) => {
```

Comment:
REQ: This is kind of silly and opinionated, but could we change this to `isNotFunc`? 

Reasons: 
- `isNot` is known to our codebase while `isnt` isn't. Keeping `isNot` will maintain consistency which will help with readibility
- the missing apostrophe may throw someone off one day, especially if English is not their first language

Reply:
agreed, will fix.